backend/tests/validation/test_suite_integrity.py

```
@@ -0,0 +1,675 @@
+    async def test_slack_bench_test_12_via_api(self, slack_client_with_differ):
+        """test_12: Edit message via API."""
+        client = slack_client_with_differ["client"]
+        differ = slack_client_with_differ["differ"]
+
+        suite_file = EXAMPLES_DIR / "slack" / "testsuites" / "slack_bench.json"
+        with open(suite_file) as f:
+            suite = json.load(f)
+
+        # Note: test_12 impersonates U02JOHNDOE1, but slack_client uses U01AGENBOT9
+        # We need to use the AGENT's message for this test
+        test_spec = next(t for t in suite["tests"] if t["id"] == "test_12")
```

Ruff (0.14.2)

Local variable `test_spec` is assigned to but never used

Remove assignment to unused variable `test_spec`

(F841)